client/src/Pages/SearchPage.js

```diff
@@ -0,0 +1,208 @@
+import React from "react";
+import { Form, CardDeck, InputGroup, FormControl, Card, Button, Dropdown, DropdownButton } from "react-bootstrap";
+
+export default class SearchPage extends React.Component {
+
+  constructor() {
+    super();
+    
+    this.state = {
+        dropDownValue: "Search By Categories",
+        placeholder: "Search Keywords",
+        searchText: ""
+    }
+  }
+    
+  changeValue(text, holder) {
+    this.setState({dropDownValue: text, placeholder: holder})
+  }
+
+  handleroute = routes => () => {
+    this.props.history.push({ pathname: routes });
+  };
+
+  handleSearchInput = event => {
+    this.setState({
+      searchText: event.target.value
+    });
+  };
+
+  handleSearchSubmit = () => {
+    if (this.state.searchText) {
+      this.props.history.push({
+        pathname: "/results",
+        state: {
+          searchText: this.state.searchText
+        }
+      });
+    } else {
+      alert("Please enter some search text!");
+    }
+  };
+
+  render() {
+    return (
+        <div style={{ "text-align": "center", "margin": "50px 50px" }}>
+        <Card style={{"box-shadow": "0 4px 8px 0 rgba(0,0,0,0.2)", "margin-bottom": "50px"}}>
+          
+        
+            <Card.Header><h1>Search for companies, job positions, or career fairs</h1></Card.Header>
+        <InputGroup className="mb-3">
+            <DropdownButton
+                as={InputGroup.Prepend}
+                variant="outline-secondary"
+                title={this.state.dropDownValue}
+                id="input-group-dropdown-1"
+            >
+                <Dropdown.Item as="button"><div onClick={(e) => this.changeValue(e.target.textContent, "Search Keywords")}>Search By Company</div></Dropdown.Item>
+                <Dropdown.Item as="button"><div onClick={(e) => this.changeValue(e.target.textContent, "Search Keywords")}>Search By Position</div></Dropdown.Item>
+                <Dropdown.Item as="button"><div onClick={(e) => this.changeValue(e.target.textContent, "date")}>Search By Career Fair</div></Dropdown.Item>
+                
+            </DropdownButton>
+            
+            <FormControl type={this.state.placeholder} onChange={this.handleSearchInput}
+              value={this.state.searchText} placeholder="Search Keywords" aria-describedby="basic-addon1" />
+            <InputGroup.Append>
+                <Button variant="primary" type="submit" id="basic-addon2" onClick={this.handleSearchSubmit}>Search</Button>
+            </InputGroup.Append>
+        </InputGroup>
+        
+        </Card>
+        
+
+        <CardDeck>
+            <Card style={{"box-shadow": "0 4px 8px 0 rgba(0,0,0,0.2)"}}>
+                <div style={{ "width": "200px", "margin": "auto"}}> 
+                <Card.Img variant="top" src="holder.js/171x180" height="200" />
+                </div>
```

Comment:
Would it be possible for this to vary based on the dropdown selection?
That is if you select search by career fair in the drop down, then this shows all career fair cards, 
if positions, then it shows all jobs cards etc.